selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsPalette.java

```diff
@@ -158,11 +158,13 @@ public void commandIsNotExists(String commandName) {
    * @param commandName name of the command
    */
   public void startCommandByDoubleClick(String commandName) {
-    actions
-        .doubleClick(
-            seleniumWebDriver.findElement(By.xpath(String.format(Locators.COMMANDS, commandName))))
-        .build()
-        .perform();
+    String locatorToCurrentCommand = String.format(Locators.COMMANDS, commandName);
```

Comment:
Let's use static import of **String.format()** method to make code a little bit clear.